frontend/src/apiAccess/apis.js

```diff
@@ -379,6 +379,31 @@ async function activityLogDayCountsPage({
   return { response, responseData };
 }
 
+async function activityLogEntries({
+  eventTypeName,
+  effectedUser,
+  actorUser,
+  page,
+}) {
+  const limit = 20;
+  const offset = calculateOffset({ page, limit });
+  let params = {
+    limit,
+    offset,
+  };
+
+  if (eventTypeName) params["event_type__name"] = eventTypeName;
+  if (actorUser) params["actor_user"] = actorUser;
+  if (effectedUser) params["effected_user"] = effectedUser;
```

Comment:
There are a few missing filters here: 

This is how it is set up in the backend
```
filterset_fields = [
        "event_type",
        "actor_user",
        "effected_user",
        "object_1_content_type",
        "object_1_id",
        "object_2_content_type",
        "object_2_id",
    ]
```

Ideally we should be able to make use of all the filters.